src/main/java/bio/terra/cli/utils/JacksonMapper.java

```diff
@@ -25,7 +26,7 @@
 
   /** Getter for the singleton instance of the default Jackson {@link ObjectMapper} instance. */
   public static ObjectMapper getMapper() {
-    return objectMapper;
+    return objectMapper.configure(JsonParser.Feature.ALLOW_COMMENTS, true);
```

Comment:
I didn't know about this config, would have been really helpful. Thanks for the pointer!